examples/experimental/ProductionMonitoring/StreamTable.md

```diff
@@ -0,0 +1,94 @@
+# Weave StreamTable
+
+Log and explore some basic StreamTables now in the [interactive notebook version](../ProductionMontoring/stream_table_api.ipynb) of these docs.
+
+A Weave StreamTable object enables continuous streaming of data from an application or service to W&B. This is an extension of the standard wandb.Table object to handle monitoring use cases. Instead of uploading a complete, immutable Table object once, you can append data repeatedly to the same StreamTable object with `.log([your data rows])`.
```

Comment:
Do you mind removing: `This is an extension of the standard wandb.Table object to handle monitoring use cases. Instead of uploading a complete, immutable Table object once,`

Reply:
removed, maybe there's a future/hypothetical spot for contextualizing StreamTable usage relative to "regular" Tables. Unless all Tables are going to be StreamTables :)